### DISCUSSION TITLE
How to apply it for widgets?

### DISCUSSION THREAD

**Discussion by RuthvikSaiKumar** [rdbende/Sun-Valley-ttk-theme#60] [Q&A] [answered]
Hi,
First of all, this is an amazing thing.

I just want to know, if I should do any extra steps to add the theme to buttons, checkbuttons etc, because, light and dark theme is working but the widgets are just having the same default theme.

Thank you.

**Comment by rdbende** [comment, marked as answer]
Hi!
Thanks for using my theme, I'm glad you like it!

Seems like you're using plain Tkinter widgets, not `ttk` ones. In order to apply the theme to the widgets, you need to use `ttk` (themed tk) widgets. It's as simple as just importing `ttk` from tkinter and then replacing the plain tkinter widgets with their themed counterpart.

Like this:
```py
from tkinter import ttk

button = ttk.Button(...)
```

**Reply by RuthvikSaiKumar** [reply]
Ok thank you, it works!
Didn't explore all those 😅
